src/flownet/config_parser/_config_parser.py

```diff
@@ -209,6 +213,63 @@ def _to_abs_path(path: Optional[str]) -> str:
                             "concave_hull": {MK.Type: types.Bool, MK.AllowNone: True},
                         },
                     },
+                    "constraining": {
+                        MK.Type: types.NamedDict,
+                        MK.Content: {
+                            "kriging": {
+                                MK.Type: types.NamedDict,
+                                MK.Content: {
+                                    "enabled": {
+                                        MK.Type: types.Bool,
+                                        MK.Default: False,
+                                        MK.Description: "Switch to enable or disable kriging on well log data.",
+                                    },
+                                    "n": {
+                                        MK.Type: types.Integer,
+                                        MK.Default: 20,
+                                        MK.Description: "Number of kriged values in each direct. E.g, n = 10 -> "
+                                        "10x10x10 = 1000 values. Default: 20",
```

Comment:
If default values are not automatically shown with `ConfigSuite` we should make a feature request there. If it is, or will be in near future, we could consider removing `Default: `?

Reply:
Default values are not shown in our current documentation. I agree with what you are saying. Will remove and make an issue.

Reply:
Two issues already existed on that, one a few weeks old: https://github.com/equinor/configsuite/issues/164

Guess they have it on the radar so we can drop the written defaults.